### DISCUSSION TITLE
The styles is invalid after build, could someone help me?

### DISCUSSION THREAD

**Discussion by SnowingFox** [nativewind/nativewind#416] [Q&A] [answered]
I'm using expo to build app, but i met a problem

In expo dev mode, `nativewind` works well
<img src="https://user-images.githubusercontent.com/26035718/218256696-69eb3275-a50f-48c8-a84c-b63e914bc30c.jpg" width="300px" height="700px" />

But after building through expo, the styles are all invalid, i don't know what happen and how to solve it, could someone help me?
<img src="https://user-images.githubusercontent.com/26035718/218256688-65d8e5d5-5b55-4053-9a11-9b7c4f132aed.jpg" width="300px" height="700px" />

This is my project https://github.com/HFUTHole/HFUTHoleApp

`babel.config.js`
```js
module.exports = function (api) {
  api.cache(true)

  const plugins = [
    ['nativewind/babel', { mode: 'transformOnly' }],
    [
      'module-resolver',
      {
        root: ['.'],
        alias: {
          '@': './src',
        },
        extensions: ['.js', '.jsx', '.ts', '.tsx'],
      },
    ],
  ]

  return {
    presets: ['babel-preset-expo'],
    env: {
      production: {
        plugins: ['react-native-paper/babel', ...plugins],
      },
    },
    plugins,
  }
}
```

`tailwind.config.js`
```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./App.tsx', './src/**/*.{js,jsx,ts,tsx}'],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

**Comment by SnowingFox** [comment]
I also run ` npx tailwind -o output.css`, i'm sure all className are matched

**Comment by SnowingFox** [comment]
i was using `eas build -p android --profile preview` to build my app

**Comment by SnowingFox** [comment, marked as answer]
i got it, replace `['nativewind/babel', { mode: 'transformOnly' }],` to `['nativewind/babel'],` that can work well

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
